src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -55,14 +61,16 @@ public function load($data, $type = null)
 
         foreach ($this->resourceNameCollectionFactory->create() as $resourceClass) {
             $resourceMetadata = $this->resourceMetadataFactory->create($resourceClass);
-            $normalizedShortName = Inflector::pluralize(Inflector::tableize($resourceMetadata->getShortName()));
+
+            $shortNamePath = $resourceMetadata->getShortName() ?? '';
```

Comment:
No, and no.

We should throw an exception here if we get `null` from `ResourceMetadata::getShortName`, because we cannot proceed normally. It's much better than generating a nonsensical route / path.